modules/apps/headless/headless-delivery/headless-delivery-impl/src/main/java/com/liferay/headless/delivery/internal/dto/v1_0/converter/DocumentDTOConverter.java

```diff
@@ -162,6 +164,20 @@ public Document toDTO(DTOConverterContext dtoConverterContext)
 				setExternalReferenceCode(fileEntry::getExternalReferenceCode);
 				setFileExtension(fileEntry::getExtension);
 				setFileName(fileEntry::getFileName);
+				setFriendlyUrlPath(
+					() -> {
+						FriendlyURLEntry friendlyURLEntry =
```

Comment:
@Mikellorza, in the story the display page is mentioned. So just wondering which URL we should expose

Reply:
But, to double check we can ask Ruth to confirm it.